### ISSUE TITLE
Does tensorflow 2.X provide functions similar to "tf.contrib.lookup.index_table_from_tensor" in TensorFlow 1.X?

### ISSUE THREAD

**Issue by mrchor** [tensorflow/tensorflow#39376]
In TensorFlow 1.X, "tf.contrib.lookup.index_table_from_tensor" is a very useful function. But I haven't found a similar function yet in TensorFlow 2X. So does tensorflow 2.X provide functions similar to "tf.contrib.lookup.index_table_from_tensor" in TensorFlow 1.X?
Thank you very much.

**Comment by Saduf2019** [comment]
@mrchor 
Could you please refer to this link and let us know if it helps:
[link](https://stackoverflow.com/questions/53383032/option-for-tf-contrib-lookup-index-table-from-tensor)
[link2](https://www.programcreek.com/python/example/90379/tensorflow.tables_initializer)

**Comment by mrchor** [comment]
Yes, these link is useful to me.
But I found that tensorflow's "index_table" does not appear to be able to train in parallel in GPU.
There will be some trouble：
![image](https://user-images.githubusercontent.com/10007145/81764598-814df880-9504-11ea-9843-3fb1093d49b6.png)
And I tested that it is normal to run without "index_table" in parallel in GPU.

**Comment by Saduf2019** [comment]
@mrchor 
As the issue reported in is addressed, could you please open a new issue for the one reported above.

**Comment by mrchor** [comment]
@Saduf2019 OK，thanks ~
